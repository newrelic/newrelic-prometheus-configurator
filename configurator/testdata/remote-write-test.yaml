data_source_name: "data-source"
newrelic_remote_write:
  license_key: nrLicenseKey
  staging: true
  extra_write_relabel_configs:
    - source_labels: ['__name__', 'instance']
      regex: node_memory_active_bytes;localhost:9100
      action: drop
  proxy_url: http://proxy.url.to.use:1234
  tls_config:
    insecure_skip_verify: true
    ca_file: /path/to/ca.crt
    key_file: /path/to/key.crt
    cert_file: /path/to/cert.crt
    server_name: server.name
    min_version: TLS12
  queue_config:
    capacity: 2500
    max_shards: 200
    min_shards: 1
    max_samples_per_send: 500
    batch_send_deadline: 5s
    min_backoff: 30ms
    max_backoff: 5s
    retry_on_http_429: false
  remote_timeout: 30s

extra_remote_write:
  - url: https://extra.prometheus.remote.write
