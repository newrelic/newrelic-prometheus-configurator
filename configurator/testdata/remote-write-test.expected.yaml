remote_write:
  - url: https://staging-metric-api.newrelic.com/prometheus/v1/write?prometheus_server=data-source
    remote_timeout: 30s
    authorization:
      credentials: nrLicenseKey
    tls_config:
      ca_file: /path/to/ca.crt
      cert_file: /path/to/cert.crt
      key_file: /path/to/key.crt
      server_name: server.name
      insecure_skip_verify: true
      min_version: TLS12
    proxy_url: http://proxy.url.to.use:1234
    queue_config:
      capacity: 2500
      max_shards: 200
      min_shards: 1
      max_samples_per_send: 500
      batch_send_deadline: 5s
      min_backoff: 30ms
      max_backoff: 5s
      retry_on_http_429: false
    write_relabel_configs:
      - action: drop
        regex: node_memory_active_bytes;localhost:9100
        source_labels:
          - __name__
          - instance
  - url: https://extra.prometheus.remote.write
